packages/sdk/src/plt/TokenEvent.ts

```diff
@@ -0,0 +1,102 @@
+import * as Proto from '../grpc-api/v2/concordium/protocol-level-tokens.js';
+import { HexString } from '../types.js';
+
+export type JSON = HexString;
```

Comment:
Maybe we should
```suggestion
export type JSON = any;
```

Reply:
it makes sense to me that the default JSON encoding is a hex string, but it sounds like a good idea to provide the cbor decode functions as well as long as the library can be tree shaken.

Reply:
This will be implemented in another PR, which will include general serialization of plt types and transactions.